src/Evolution/Systems/CurvedScalarWave/Worldtube/ElementActions/InitializeConstraintGammas.hpp

```diff
@@ -0,0 +1,51 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Tags.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
+
+namespace CurvedScalarWave::Worldtube::Initialization {
+
+/*!
+ * \ingroup InitializationGroup
+ * \brief Mutator meant to be used with
+ * `Initialization::Actions::AddSimpleTags` to initialize the constraint
+ * damping parameters of the CurvedScalarWave system according to a Gaussian
```

Comment:
Can you line/sentence break after `system`. This is very long for it to show up in Doxygen's "brief" section